torch/ao/quantization/fx/_decomposed.py

```diff
@@ -560,8 +521,8 @@ def dequantize_per_channel(
     out = res.permute(tuple(permute_axis_list))
     return out
 
-@impl(quantized_decomposed_lib, "dequantize_per_channel", "Meta")
-def dequantize_per_channel_meta(
+@dequantize_per_channel.register_fake
+def _(
```

Comment:
If I have multiple of these in the same file will it mess with the namespace?

Reply:
Nope, the `_` is just like any other name in python (e.g. `impl`)